deerlab/dipolarkernel.py

```diff
@@ -60,9 +61,15 @@ def dipolarkernel(t, r, *, pathways=None, mod=None, bg=None, method='fresnel', e
         
         The default is ``'fresnel'``.
 
+    orisel : array_like, optional 
+        Probability distribution of possible orientations of the interspin vector to account for orientation selection. Must be 
+        vector defined for uniformly distributed orientations in the range [0,π/2]. If speficied as ``None`` (by default), a uniform distribution is assumed. 
```

Comment:
This is specific to axial distributions, right? P(theta,phi) is not supported? This needs to be stated clearly. Also, describe exactly what theta is.

Reply:
Yes, it is specific to axial distributions. Although it would be easy to expand this to include `phi`.

Reply:
I think only theta matters for the DEER signal. The user should take care of integration over phi.